kubernetes/kubernetes.libsonnet

```diff
@@ -354,6 +358,8 @@ local letsbuildServiceDeployment(
       + pdb.spec.selector.withMatchLabels(dc.labels)
   ),
 
+  keda: if withKedaScaler then keda.scaledObject(ks),
```

Comment:
Don't we want to disable HPA creation if we create a scaledobject?

Reply:
Dissabled